index.js

```diff
@@ -0,0 +1,44 @@
+const books = [];
+
+function updateBookList() {
+  const bookList = document.getElementById('booksList');
+
+  bookList.innerHTML = '';
+  books.forEach((book, index) => {
+    const li = document.createElement('li');
+    li.textContent = `${book.title} by ${book.author}`;
+
+    function removeBook(index) {
+      books.splice(index, 1);
+      updateBookList();
+    }
+
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = 'Remove';
+    removeBtn.addEventListener('click', () => removeBook(index));
+
+    li.appendChild(removeBtn);
+    bookList.appendChild(li);
+  });
+}
+
+function addBook(event) {
+  event.preventDefault();
+
+  const titleInput = document.getElementById('title');
+  const authorInput = document.getElementById('author');
+  const title = titleInput.value;
+  const author = authorInput.value;
+
+  const book = { title, author };
+
+  books.push(book);
+
+  titleInput.value = '';
+  authorInput.value = '';
+
+  updateBookList();
+}
+
+const bookForm = document.getElementById('bookForm');
+bookForm.addEventListener('submit', addBook);
```

Comment:
Kindly note that as part of the requirement, you should implement LocalStorage feature for this your project, so that on page reload, the data will be preserved.

Project requirement :

![image](https://github.com/Uzair-Manzoor/Awesome-books/assets/20537712/97e1c9ec-9b38-4159-96ae-e7bb82c078bb)